frontend/src/metabase-lib/expressions/helper-text-strings.ts

```diff
@@ -730,6 +730,34 @@ const helperTextStrings: HelpText[] = [
       },
     ],
   },
+  {
+    name: "convert-timezone",
+    structure:
+      "convertTimezone(" +
+      t`column` +
+      ", " +
+      t`target` +
+      ", " +
+      t`source` +
+      ")",
```

Comment:
Since source is optional, it should have square brackets around it
```suggestion
      ", [" +
      t`source` +
      "])",
```

Reply:
Changed in 7dd21288b0